### PR TITLE
Feature: "initiallly show cards and then hide" added

### PR DESCRIPTION
**Pull request by Prakhar1106**
issue #8 fixed

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,6 +52,23 @@ function toggleSound(){
         timer.innerHTML = "<b>" + counter + "</b>";
     }, 1000);
 
+    // refresh when reload
+    const showAllCards = ()=>{
+        cards.forEach((card, i)=>{
+            card.children[0].src=cardImageSrcs[i]; // Setting image source for flipped front face
```
Comment (dev-lovedeep) [inline index.js]: `cardImageSrc` contain all the images , but the actual images are shown using `cardArray`
this Is causing a bug that the image you are showing are not in order what actually in the game

File: index.js [hunk 2]
```diff
@@ -52,6 +52,23 @@ function toggleSound(){
...
+            card.children[0].alt=cardImageSrcs[i].split('/').slice(-1)[0].split('.').slice(0, -1).join('.'); // Setting image file name as alt text for flipped front face
+            card.children[1].style.display="none";
+        })
+        
+        setTimeout(()=>{
+            cards.forEach((card)=>{
+                card.children[0].src="#"; // Removing image src so that it isn't visible through HTML
+                card.children[0].alt="card front face"; // Removing image alt so that it isn't visible through HTML
+                card.children[1].style.display="block";
```
Comment (dev-lovedeep) [inline index.js]: this same logic is used in the flip card function. So just remove the Repeatation

**Review by dev-lovedeep — COMMENTED** [review]
fix following issues

File: index.js (hunk 1 above)
Comment (Prakhar1106) [reply index.js]: this issue I already resolved, in updated code
https://github.com/ContriHUB/memory-game/pull/26/commits/e77281f72f25c03c8ce50ab3403775ce7b58a8da

File: index.js [hunk 3]
```diff
@@ -52,6 +52,23 @@ function toggleSound(){
...
+            })
+            counter = 0;
+        }, 5000);
```
[the remark this answers is not in the document]
Comment (Prakhar1106) [reply index.js]: Okay

File: index.js (hunk 2 above)
Comment (Prakhar1106) [reply index.js]: Okay, I'll create separate function for this

**Comment by Prakhar1106** [comment]
Please review, I implemented suggested changes

File: index.js [hunk 4]
```diff
@@ -100,8 +127,11 @@ function toggleSound(){
         return array;
     }
     // Shuffling cards
-    shuffle(cardArray);
+    shuffle(cardImageSrcs);
```
Comment (dev-lovedeep) [inline index.js:130]: this must be `cardArray`

**Review by dev-lovedeep — COMMENTED** [review]
LGTM! but commit messages like "implemented suggested changes" are not good.

**Comment by dev-lovedeep** [comment]
also resolve the conflicts

File: index.js (hunk 4 above)
Comment (Prakhar1106) [reply index.js]: changed!!

**Comment by Prakhar1106** [comment]
done!!

**Pull request merged**